screens/Contact/ContactDetailScreen.js

```diff
@@ -984,7 +984,7 @@ class ContactDetailScreen extends React.Component {
           </Col>
           <Col>
             <Text style={{ marginTop: 'auto', marginBottom: 'auto' }}>
-              {this.state.contact.subassigned ? this.state.contact.subassigned.values.map(contact => this.state.usersContacts.find(user => user.value === contact.value).name).join(', ') + (this.state.contact.subassigned.values.length > 0 ? '.' : '') : ''}
+              {this.state.contact.subassigned ? this.state.contact.subassigned.values.map(contact => this.state.usersContacts.find(user => user.value === contact.value).name).join(', ') + (this.state.contact.subassigned.values.length > 0 ? '' : '') : ''}
```

Comment:
That whole ternary statement can just be remove since neither the true nor false condition add anything but an empty string.

Reply:
I have done as requested. All the extra length checks have been removed in the 2 files.